src/index.ts

```diff
@@ -87,6 +87,8 @@ export default new Vue({
     } else if (window.ethereum) { // modern MetaMask-like wallet
       // @ts-ignore
       const ethereum = window.ethereum
+      // walletlink steals typing of window.ethereum
+      // @ts-ignore
```

Comment:
Hmm, so actually... I don't think we want any of this auto-select wallet stuff to run at all when `features.bscWallets === true`

Reply:
Let's make deal with that in another PR

Reply:
removed